apps/glusterfs/device_entry.go

```diff
@@ -161,14 +161,11 @@ func (d *DeviceEntry) ConflictString() string {
 	return fmt.Sprintf("Unable to delete device [%v] because it contains bricks", d.Info.Id)
 }
 
-func (d *DeviceEntry) Delete(tx *bolt.Tx) error {
-	godbc.Require(tx != nil)
-
+func (d *DeviceEntry) CheckDelete(tx *bolt.Tx) error {
```

Comment:
If CheckDelete doesn't use tx it should not take it as an argument.

Reply:
missed that one :( fixed now, PTAL.